src/web/style.css

```
@@ -1460,6 +1631,186 @@ footer a:hover {
+  position: relative;
+  z-index: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding: 2rem;
+  max-width: 640px;
+  animation: splashFadeIn 0.6s ease-out;
+}
+
+@keyframes splashFadeIn {
```

Stylelint (17.7.0)

[error] Expected keyframe name "splashFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)